#!/usr/bin/env python
PACKAGE = "lwr_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("damp_0_joint", double_t,    0, "damping parameter joint 0", 0,  0, 100)
gen.add("damp_1_joint", double_t,    0, "damping parameter joint 0", 0,  0, 100)
gen.add("damp_2_joint", double_t,    0, "damping parameter joint 0", 0,  0, 100)
gen.add("damp_3_joint", double_t,    0, "damping parameter joint 0", 0,  0, 100)
gen.add("damp_4_joint", double_t,    0, "damping parameter joint 0", 0,  0, 100)
gen.add("damp_5_joint", double_t,    0, "damping parameter joint 0", 0,  0, 100)
gen.add("damp_6_joint", double_t,    0, "damping parameter joint 0", 0
,  0, 100)

exit(gen.generate(PACKAGE, "dynamic_lwr_damping", "damping_param"))
